lrpar/src/lib/ctbuilder.rs

```diff
@@ -475,6 +564,8 @@ where
 
         let mut f = File::create(outp_rs)?;
         f.write_all(outs.as_bytes())?;
+
+        self.cdata = Some((grm, sgraph, stable));
```

Comment:
Maybe it would make more sense to have something like:

```rust
struct CTParserBuilder {
 ...
 conflicts: Option<(YaccGrammar, StateGraph, StateTable, Conflicts)>
```

and this line would be:

```rust
if let Some(c) = stable.conflicts() {
    self.conflicts = Some((grm, sgraph, stable, c));
}
```

and then `fn conflicts` would just  be `self.conflicts`.

Reply:
Since `c` is a reference, I would have to add lifetimes, which isn't really worth it for a private variable that can only be accessed via `fn conflicts` anyways. So I just renamed `cdata` to `conflicts`.

Reply:
Fair enough.